vendor/vendor.json

```diff
@@ -0,0 +1,13 @@
+{
+	"comment": "",
+	"ignore": "",
+	"package": [
+		{
+			"checksumSHA1": "MvXDQF38CGswwuUsgdethojyXfc=",
+			"path": "github.com/Microsoft/hcsshim",
+			"revision": "b3f49c06ffaeef24d09c6c08ec8ec8425a0303e2",
+			"revisionTime": "2019-11-14T18:46:01Z"
+		}
+	],
+	"rootPath": "github.com/wawa0210/containerd"
+}
```

Comment:
seems unrelated

Reply:
I do n’t know how to use vendors, I ’m so sorry